ackermann_steering_controller/CMakeLists.txt

```diff
@@ -0,0 +1,107 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(ackermann_steering_controller)
+
+find_package(catkin REQUIRED COMPONENTS
+  angles
+  control_toolbox
+  controller_interface
+  diff_drive_controller
+  gazebo_ros_control
```

Comment:
not used at al (?)

Reply:
same for angles,  control_toolbox and  joint_limits_interface

Reply:
Fix at https://github.com/ros-controls/ros_controllers/pull/356/commits/defd9a17a40d824a2818d808e898e3131badd466